evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -266,6 +266,15 @@ contract Operator is
     return address(s_LinkToken);
   }
 
+  function foreward(address _to, bytes calldata _data)
+    public
+    onlyAuthorizedNode()
+  {
+    require(_to != address(s_LinkToken), "Cannot use #foreward to send messages to Link token");
```

Comment:
gas optimization: make s_LinkToken `immutable`.

Reply:
I'm in favor of making it immutable, but I think the convention would be to remove the `s_` prefix as well.

Reply:
sgtm